recipes/newspaper3k/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "newspaper3k" %}
+{% set version = "0.2.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "abe2ff199e5a41cc9d690fd5ade7cf605dc542f98b1071b26e91ef136a50042d" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```

Comment:
It appears that only `python` and `setuptools` are required to build the package since we're using `--single-version-externally-managed`. Please remove all others which are only required at runtime.